src/axom/primal/operators/compute_moments.hpp

```diff
@@ -66,6 +69,9 @@ T sector_area(const primal::BezierCurve<T, 2>& curve)
 template <typename T>
```

Comment:
Please also add a note to the doxygen for these functions that they only work on rational Bezier curves.

Reply:
Actually, perhaps we should implement the `area()` and `centroid()` for rational polygons  in terms of `evaluate_integral` ?
Can you please create an issue to track this?

Reply:
I've added this to issue #897, as I agree that it should work in the more general case. That being said, the current methods for nonrational polygons will be significantly faster than invoking quadrature, and won't require the MFEM include. I think a thorough implementation would proceed largely as before, and only compute areas/centroids of sectors using quadrature when necessary, rather than just broadly to the entire polygon.